MIGRATIONS.md

```diff
@@ -105,9 +105,19 @@ The new migration should contain a class called `Migration` which inherits from
 
 ### Adding an SQL migration (ClickhouseNodeMigration)
 
-A ClickhouseNodeMigration is the most common type of migration and determines the SQL to be run on each ClickHouse node. You should define all four methods - `forwards_local`, `backwards_local`, `forwards_dist` and `backwards_dist` in order to provide the DDL for all ClickHouse layouts that a user may have. The operations provided in the `_local` methods will run on each local ClickHouse node and the `_dist` methods will run on each distributed ClickHouse nodes (if any).
+A ClickhouseNodeMigration is the most common type of migration and determines the SQL to be run on each ClickHouse node. You should define all two methods - `forwards_ops` and `backwards_ops` in order to provide the DDL for all ClickHouse layouts that a user may have. These functions return a sequence of sql ops for the forwards and backwards (reverse) migrations.
```

Comment:
`You should define all two methods ...` -> `You must define two methods...`  (I know this changes a little bit of how it was written before but I think must is better than should here)

Reply:
fixed